docs/_includes/xref-source-to-source.adoc

```diff
@@ -21,3 +21,17 @@ We could also write the link as link:README{outfilesuffix}[README].
 The links in the generated document will now point to [path]_README.adoc_ instead of the default, [path]_README.html_.
 
 CAUTION: You probably don't want to set `outfilesuffix` to `.adoc` without the `ifdef` condition as it could result in Asciidoctor overwriting input files when you run it locally (though there's some protection against this).
+
+The `relfileprefix` attribute is prefixed to relative paths, and is often used with `outfilesuffix`.
```

Comment:
We should clarify here that it is prefixed to relative xref paths. It is not prefixed to all relative paths (for instance, it is not applicable for relative paths defined using the link macro).